script/setup/install-runc

```diff
@@ -21,13 +21,14 @@
 set -eu -o pipefail
 
 function install_runc() {
-	RUNC_COMMIT=$(grep opencontainers/runc "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')
+	# When updating RUNC_VERSION, consider updating the runc module in go.mod as well
+	: "${RUNC_VERSION:=v1.0.0-rc93}"
```

Comment:
could we please pick up the RUNC_VERSION from a file? ( we could have a file name RUNC_VERSION with the contents of `v1.0.0-rc93` ) So it will then be easy to `curl` the version from a branch or master (directly from github raw url) easily for automation (in downstream or other CI in other projects).

Reply:
So; pick from file, but allow the `RUNC_VERSION` env-var to override, correct? Yes I guess we can do that. Let me have a look

Reply:
works! thanks!